data/config/example.ini

```
@@ -23,4 +23,7 @@ step: 0.1
 [TIME]
 min: -1
 max: 1
-step: 1
\ No newline at end of file
+step: 1
+
+[DETECTOR]
+geomscope: IC86
```

I think this field should be renamed to be consistent with the argument of `asteria.simulation.Simulation`
```suggestion
detector_scope: IC86
```

---

This seems like a reasonable suggestion, though I'll need to revert the changes in the `gen-sim` branch temporarily in order to eliminate merge conflicts with `main`.